client/src/app/site/motions/models/view-category.ts

```diff
@@ -0,0 +1,124 @@
+import { Category } from '../../../shared/models/motions/category';
+import { TranslateService } from '@ngx-translate/core';
+import { BaseModel } from '../../../shared/models/base/base-model';
+import { BaseViewModel } from '../../base/base-view-model';
+
+/**
+ * Category class for the View
+ *
+ * Stores a Category including all (implicit) references
+ * Provides "safe" access to variables and functions in {@link Category}
+ * @ignore
+ */
+export class ViewCategory extends BaseViewModel {
+    private _category: Category;
+    private _id: number;
```

Comment:
usually _id and _name do not need to be declared here, since it's rare to actually change the name of an existing view category. We usually just submit the forms values with the update process and no altered object(s).